<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTransmitDlg</class>
 <widget class="QDialog" name="CTransmitDlg">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transmit Setup</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QSlider" name="LowShelfGainSlider">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>22</width>
     <height>121</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-25</number>
   </property>
   <property name="maximum">
    <number>25</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSlider" name="LowShelfFreqSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>230</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Low Shelf</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="LowGainEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LowFreqEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PkGainEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QSlider" name="PkFreqSlider">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>30</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>15</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>2</number>
   </property>
  </widget>
  <widget class="QSlider" name="PkGainSlider">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>22</width>
     <height>121</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-25</number>
   </property>
   <property name="maximum">
    <number>25</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Peak</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="PkFreqEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="HiGainEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QSlider" name="HiShelfFreqSlider">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>30</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>2</number>
   </property>
  </widget>
  <widget class="QSlider" name="HiShelfGainSlider">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>22</width>
     <height>121</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-25</number>
   </property>
   <property name="maximum">
    <number>25</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>High Shelf</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="HiFreqEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxButtons">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>121</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Test Signals</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="TestOffButton">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Test Sig Off</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="TestNoiseButton">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Test Noise</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="TestToneButton">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Test Tone</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ToneEdit">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="ToneFreqSlider">
      <property name="minimum">
       <number>100</number>
      </property>
      <property name="maximum">
       <number>3000</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>100</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>200</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="FlatButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Preset Flat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Recall1Button">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Recall 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Recall2Button">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Recall 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Save1Button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>200</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Save2Button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>230</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save 2</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="CtcssSpinBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="suffix">
    <string> Hz</string>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>255.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FM CTCSS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>TestOffButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>CTransmitDlg</receiver>
   <slot>OnTestOffButton(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TestNoiseButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>CTransmitDlg</receiver>
   <slot>OnTestNoiseButton(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CTransmitDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CTransmitDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TestToneButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>CTransmitDlg</receiver>
   <slot>OnTestToneButton(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnTestOffButton(bool)</slot>
  <slot>OnTestNoiseButton(bool)</slot>
  <slot>OnTestToneButton(bool)</slot>
 </slots>
</ui>
